src/codewhisperer/util/runtimeLanguageContext.ts

```diff
@@ -161,7 +161,22 @@ export class RuntimeLanguageContext {
      * @param languageId : vscode language id or codewhisperer language name
```

Comment:
please update this jsdoc with the new parameter

Reply:
Updated!